src/GraphRequestUtil.ts

```diff
@@ -58,3 +58,99 @@ export const serializeContent = (content: any): any => {
 	}
 	return content;
 };
+
+export class PathUtils {
+	private static hostRe = /([a-z]+.microsoft.[a-z]+)/g;
+	private static versionRe = /(beta|v1.0)/g;
```

Comment:
What do you think about not hardcoding the versions? My concern is that we shouldn't miss updating this when we have newer versions.

Reply:
I think this is fine as the versions don't change often.